plugin/src/main/java/org/opensearch/ml/action/models/DeleteModelTransportAction.java

```diff
@@ -181,6 +164,31 @@ protected void doExecute(Task task, ActionRequest request, ActionListener<Delete
         }
     }
 
+    @VisibleForTesting
+    void retryDeleteModel(
+        String modelId,
+        MLModelState mlModelState,
+        Boolean isHidden,
+        MLModelDeleteRequest mlModelDeleteRequest,
+        ActionListener<DeleteResponse> actionListener
+    ) {
+        Boolean retry = mlModelDeleteRequest.getRetry();
```

Comment:
where exactly are these retry fields being used?

Reply:
At that time the PR is not ready to be reviewed yet. Now is ok.